### PR TITLE
Update Readme Directory

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
# What does this PR do?
Update Readme Directory

### CODE REVIEW

**Comment by sean-m-sullivan** [comment]
image

File: README.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ This Ansible collection allows for easy interaction with an Ansible Automation H
...
-## Links to other Validated Configuration Collections
+## Links to other Validated Configuration Collections for Ansible Automation Platform
 
 |                                      Collection Name                                       |                 Purpose                  |
 |:------------------------------------------------------------------------------------------:|:----------------------------------------:|
-| [Controller Configuration](https://galaxy.ansible.com/redhat_cop/controller_configuration) |   Automation controller configuration    |
+| [Controller Configuration](https://github.com/redhat-cop/controller_configuration) |   Automation controller configuration    |
 |             [EE Utilities](https://galaxy.ansible.com/redhat_cop/ee_utilities)             | Execution Environment creation utilities |
-|     [AAP installation Utilities](https://galaxy.ansible.com/redhat_cop/aap_utilities)      |  Ansible Automation Platform Utilities   |
+|     [AAP installation Utilities](https://github.com/redhat-cop/ee_utilities)      |  Ansible Automation Platform Utilities   |
```
Comment (djdanielsson) [inline README.md]: should be aap_utilities not ee

**Review by djdanielsson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sean-m-sullivan) [reply README.md]: Updated.

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**